resources/PythonPip3Dsc/PythonPip3Dsc.psm1

```diff
@@ -135,7 +205,7 @@ function Invoke-Pip3Uninstall {
     $command.Add((Get-PackageNameWithVersion @PSBoundParameters))
     $command.Add($Arguments)
 
-    # '--yes' is needed to ignore confirmation required for uninstalls
+    # '--yes' is needed to ignore conformation required for uninstalls
```

Comment:
`confirmation` is the correct word - might need to be added to `expect/generic_terms.txt`

Reply:
no... `confirmation` should _really_ be _in_ the dictionary. Something's going wrong here...

Here's a very simple repository with just the workflow from this repository and this excerpt:
https://github.com/check-spelling-sandbox/expert-octo-tribble/actions/runs/11696140617/job/32572710923

it's perfectly happy with `confirmation` (it's an old word) and is only unhappy about `uninstalls` (which is a relatively new word).

Reply:
I don't understand what's going on here...  Fwiw, I tried to [reproduce](https://github.com/check-spelling-sandbox/winget-dsc/actions/runs/11706782133) and at least there, check-spelling complained about `confrimation` and I agree with it's complaint.

What I see using git blame is:

https://github.com/microsoft/winget-dsc/blob/d816dc3cc0e5b82f9c2ea7e42f48e7f10c473261/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L208

https://github.com/microsoft/winget-dsc/blob/ab75bd84154e12ae1e0682f357bf229b8272b8ed/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L208

```sh
# '--yes' is needed to ignore confrimation required for uninstalls 
```

Note it says `confrimation` not `confirmation`.

This feels like a bug in git or github's handling of merging.

Steps:
1. Click the [view file](https://github.com/microsoft/winget-dsc/blob/6370252f94c386829730170a24680849545c46f6/resources/PythonPip3Dsc/PythonPip3Dsc.psm1) link here...
2. Click [blame](https://github.com/microsoft/winget-dsc/blame/6370252f94c386829730170a24680849545c46f6/resources/PythonPip3Dsc/PythonPip3Dsc.psm1)
3. Click into the source area and use cmd-f (or control-f if you aren't on macOS...)
4. Search for `required for uninstalls`
5. Click on the [line](https://github.com/microsoft/winget-dsc/blame/6370252f94c386829730170a24680849545c46f6/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L208)
   >     # '--yes' is needed to ignore conformation required for uninstalls
6. Click the before button [Blame prior to change cd15ed2, made on Nov 5, 2024](https://github.com/microsoft/winget-dsc/blame/d816dc3cc0e5b82f9c2ea7e42f48e7f10c473261/resources/PythonPip3Dsc/PythonPip3Dsc.psm1)
7. See https://github.com/microsoft/winget-dsc/blame/d816dc3cc0e5b82f9c2ea7e42f48e7f10c473261/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L208
    >     # '--yes' is needed to ignore confrimation required for uninstalls
8. Click [code](https://github.com/microsoft/winget-dsc/blob/d816dc3cc0e5b82f9c2ea7e42f48e7f10c473261/resources/PythonPip3Dsc/PythonPip3Dsc.psm1)
9. See https://github.com/microsoft/winget-dsc/blob/d816dc3cc0e5b82f9c2ea7e42f48e7f10c473261/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L208

That said, this was supposed to be fixed by: https://github.com/microsoft/winget-dsc/pull/110/commits/e72504a9b5f1bf86c52b4c577a18102f1e361a7a
https://github.com/microsoft/winget-dsc/blob/1970b700482b76b6733f0d17efd510cad02a6831/resources/PythonPip3Dsc/PythonPip3Dsc.psm1#L165

The problem came from the merge ba972039d67b897be8cf32cf24c163af00b24303
https://github.com/microsoft/winget-dsc/commit/ba972039d67b897be8cf32cf24c163af00b24303#diff-9b511215698cab300d0926d5ccdcf970247f162941926ee3de1223884753e9eeR153

Reply:
I just realized that the merge I'm fingering isn't _in_ this repository, it's in @Gijsreyn's.

My general advice to people is to always use rebase instead of merging because merging results in confusing things like this.

Reply:
Yes I'll try to think about it. Thanks Josh for the detailed inspection and explanation.

Reply:
For my own reference, I filed https://support.github.com/ticket/personal/0/3086847 about GitHub's blame/etc. here since I'm fairly certain _something_ about what it did was wrong.

Reply:
Is it possible that it was whitespace only changes?

Reply:
I'll have to talk to a git expert, I'm fairly certain it has something to do with the merge resolution and that everything did the wrong thing. I'm trying to be lazy and hoping a github staff person will find a github dev who works on git to figure this out instead of spending more time on it. All of the things I'm touching are giving me headaches 😢.